Kernel/Devices/SerialDevice.cpp

```diff
@@ -9,7 +10,7 @@
 
 namespace Kernel {
 
-UNMAP_AFTER_INIT SerialDevice::SerialDevice(int base_addr, unsigned minor)
+UNMAP_AFTER_INIT SerialDevice::SerialDevice(u32 base_addr, unsigned minor)
```

Comment:
Instead of using naked `u32`, let's utilize `IOAddress` instead.

Reply:
Good idea, done!